realm/realm-library/src/objectServer/java/io/realm/internal/network/OkHttpAuthenticationServer.java

```diff
@@ -117,6 +127,18 @@ private static URL buildActionUrl(URL authenticationUrl, String action) {
         }
     }
 
+    private static URL buildLookupUserIdUrl(URL authenticationUrl, String action, String provider, String providerId) {
+        String authURL = authenticationUrl.toExternalForm();
+        // we need the base URL without the '/auth' part
+        String baseUrlString = authURL.substring(0, authURL.indexOf(authenticationUrl.getPath()));
```

Comment:
It sure does. We need to update all our example/tests so `Constants.AUTH_URL` will not contain `auth` anymore as part of the URL. This could be done in a separate PR I believe